cmd/scaffold.go

```diff
@@ -246,6 +246,7 @@ func (fnb *FlowNodeBuilder) EnqueueNetworkInit() {
 	fnb.Component(ConduitFactoryComponent, func(node *NodeConfig) (module.ReadyDoneAware, error) {
 		cf := conduit.NewDefaultConduitFactory()
 		fnb.ConduitFactory = cf
+		node.Logger.Info().Hex("node_id", logging.ID(node.NodeID)).Msg("default conduit factory initiated")
```

Comment:
fwiw, the `handleComponents` method in scaffold should log an info message after calling this factory callback

Reply:
yes, but we also need insight on the type of conduit factory being initialized.